.github/workflows/node-js-ci.yml

```diff
@@ -28,4 +28,16 @@ jobs:
       - run: npm install
       - run: cp env.example .env
       - run: npm run build
```

Comment:
Let's get rid of this build step too, since we aren't using the build in CI.

Reply:
See https://github.com/Seneca-CDOT/telescope/pull/1444/commits/b615e5c12d143c6138f0c4bb2bd27d82355a6eea, where I'm doing that for another PR too.